test/python/topology/test_functions.py

```diff
@@ -86,4 +86,22 @@ def check_int_strings_transform_with_drop(t):
       _hwcount += 1
       return None
    raise AssertionError() 
+ 
+def strings_multi_transform():
+   return ["mary had a little lamb", "its fleece was white as snow"]
+   
+global expected_strings_multi_transform
+expected_strings_multi_transform = ["mary", "had", "a", "little", "lamb", "its", "fleece", "was", "white", "as", "snow"]
 
+def split_words(t):
+   return t.split()
+ 
+def check_strings_multi_transform(t):
+   print("TUPLE", t)
+   global _hwcount
```

Comment:
Just curious, there are a couple places in `test_functions.py` where you use the  `_hwcount` global variable. If you don't reset `_hwcount`, wouldn't it the tests that run afterwards?